book/listings/css/8/style.css

```diff
@@ -0,0 +1,4 @@
+combobox arrow {
+  color: red;
+  -gtk-icon-source: -gtk-icontheme('format-justify-fill-symbolic');
```

Comment:
Wouldn't recommend to style things like this if possible. It's not something apps should do.